packages/wes_adapter/amazon_genomics/wes/adapters/NextflowWESAdapter.py

```diff
@@ -133,10 +133,10 @@ def query_logs_for_job(self, job_details: JobDetailTypeDef, query: str):
         query_id = self.aws_logs.start_query(
             logGroupName=self.engine_log_group,
             startTime=start_time,
-            endTime=end_time or int(time.time()),
+            endTime=end_time or int(datetime.utcnow().timestamp()),
```

Comment:
After much head scratching I discovered that `datetime.utcnow().timestamp()` gives you seconds but `startTime` is in milliseconds. You want something like: `int(math.ceil(datetime.utcnow().timestamp()) * 1000)`. `math.ceil` is needed because of a potential race condition on the timestamp in seconds being less than the timestamp in milliseconds which can contain fractions of seconds.